apis/Google.Cloud.BigQuery.DataTransfer.V1/docs/history.md

```diff
@@ -1,5 +1,18 @@
 # Version history
 
+# Version 3.0.0, released 2020-09-28
+
+- [Commit 924f350](https://github.com/googleapis/google-cloud-dotnet/commit/924f350): fix!: Remove obsolete `Datasource` code
+- [Commit 6bde7a3](https://github.com/googleapis/google-cloud-dotnet/commit/6bde7a3): docs: Regenerate all APIs with service comments in client documentation
+- [Commit f83bdf1](https://github.com/googleapis/google-cloud-dotnet/commit/f83bdf1): fix: Regenerate all APIs with improved timeout behavior
+- [Commit 947a573](https://github.com/googleapis/google-cloud-dotnet/commit/947a573): docs: Regenerate all clients with more explicit documentation
+- [Commit 0c88ce0](https://github.com/googleapis/google-cloud-dotnet/commit/0c88ce0): feat: Add a new AuthorizationType for Data Source Definition.
+
+Additionally, this release included the regeneration of protobuf and gRPC types using new tooling.
+This introduces a change that has a very small chance of breaking customers. Please see
+the [FAQ entry](https://googleapis.github.io/google-cloud-dotnet/docs/faq.html#why-have-the generated-grpc-client-constructors-changed)
+for more details.
```

Comment:
We probably don't need this here, since we are bumping major version. Just another normal breaking change entry with "removed this constructor added this other should do".

Reply:
Okay, I can do that. (Presumably we can remove the FAQ entry completely?)

Reply:
Will need to wait for current task to complete before doing that though... will probably be 10 minutes.